src/msw/uilocale.cpp

```diff
@@ -32,6 +32,10 @@
     #define LOCALE_NAME_USER_DEFAULT NULL
 #endif
 
+#ifndef LOCALE_NAME_MAX_LENGTH
+#define LOCALE_NAME_MAX_LENGTH   85
```

Comment:
Very minor, but for consistency we should do
```suggestion
    #define LOCALE_NAME_MAX_LENGTH   85
```